[Cobol85] fix preprocessor compilation errors

hanohrs opened the pull request:
- Fixed compile errors of the Cobol85 preprocessor
- Added Cobol85 as a module
- Configured the test suite to run on the Cobol85 example1.txt

kaby76 commented:
Testing is broken. We get `System.Exception: Can't figure out the grammar name`, which means that we have multiple top-level .g4's. The desc.xml needs to be fixed.

hanohrs commented:
I'm sorry for breaking the tests. I had run mvn test, but not trgen.

I've reverted the line `<grammar-name>Cobol85</grammar-name>`. I had removed it earlier because desc.xml is inconsistent with _scripts/desc.xsd. It seems that desc.xsd should include the line `<xs:element name="grammar-name" type="xs:string"/>`.

kaby76 commented:
@hanohrs Thank you for bringing up this important issue regarding the desc.xsd. The file is has not been updated with more recent changes to trgen. I will update the file. https://github.com/antlr/grammars-v4/issues/4516

teverett commented:
@hanohrs  thanks!
